src/components/Search.component.jsx

```diff
@@ -0,0 +1,169 @@
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                strokeWidth="1.5"
+                stroke="#EBEBEB"
+                class="pointer-events-none w-6 h-6 absolute top-1/2 transform -translate-y-1/2 right-3"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+                />
+              </svg>
+              <input
+                className="bg-primary-color hover:bg-primary-hover text-primary-buttonText m-1 p-2 text-sm rounded-xl placeholder-primary-buttonText focus:outline-none"
+                placeholder="Search"
+                type="search"
+                {...props}
+              />
+            </label>
+          );
+        case "md":
+          return (
+            <label htmlFor="search" className="relative">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                strokeWidth="1.5"
+                stroke="#EBEBEB"
+                class="pointer-events-none w-8 h-8 absolute top-1/2 transform -translate-y-1/2 right-3"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+                />
+              </svg>
+              <input
+                className="bg-primary-color hover:bg-primary-hover text-primary-buttonText m-1 py-2 px-3 text-lg rounded-2xl placeholder-primary-buttonText focus:outline-none"
+                placeholder="Search"
+                {...props}
+              />
+            </label>
+          );
+        case "lg":
+          return (
+            <label htmlFor="search" className="relative">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="none"
+                viewBox="0 0 24 24"
+                strokeWidth="1.5"
+                stroke="#EBEBEB"
+                class="pointer-events-none w-10 h-10 absolute top-1/2 transform -translate-y-1/2 right-3"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+                />
+              </svg>
+              <input
+                className="bg-primary-color hover:bg-primary-hover text-primary-buttonText m-1 py-3 px-4 text-2xl rounded-3xl placeholder-primary-buttonText focus:outline-none"
+                placeholder="Search"
+                {...props}
+              />
+            </label>
+          );
+        default:
+          break;
+      }
+      break;
+    case "secondary":
```

Comment:
You've got text for placeholder 'white' in every size of secondary variant. I think it should be placeholder-secondary-buttonText, not placeholder-primary-buttonText.

Reply:
It was due to the color variation. The placeholder-secondary-buttonText is not visible on the secondary theme so had to go with the primary.

Reply:
<img width="217" alt="Zrzut ekranu 2023-07-2 o 12 59 08" src="https://github.com/Hack-Weekly/bronze-badger-recipe-app/assets/93224847/31e147da-c4a6-41b2-88e7-7f0706e490a0">

this how it looks for me with placeholder-secondary-buttonText

Reply:
Thought it would be confusing having same color on both the text and placeholder. 
Updated it.